hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1863,11 +1863,12 @@
             Requires `EnMasking` parameter to be enabled.
             SW randomly configures the KMAC in any hashing mode/strength, and enable EDN mode.
             Randomly enable/disable the `entropy_refresh`.
-            // TODO - error handling is not complete, do not enable `wait_timer` yet.
-            KMAC should send request to EDN once `CmdStart` is written, and should send out another
-            request to EDN when either:
+            Randomly configure `wait_timer` values (zero for disable, non-zero for timer expire).
```

Comment:
hmmm....i think i should add this to `entropy_src_edn_req` test. 
So we can make sure when there are a lot of concurrent agent nothing goes wrong.

Reply:
Would be great for stress testing EDN !

Reply:
I discussed with Tim offline and we might come up with method to check entropy request. But this looks great, thanks Eunchan